src/components/DataBar/DataBar.stories.tsx

```diff
@@ -28,7 +28,7 @@ export default {
       </div>
     ),
   ],
-} as Meta<Args>;
```

Comment:
Interesting that there's a type error if using this over `(Story: Story)` in line 21, but doesn't seem to type error for `Checkbox` or `Tooltip`

Reply:
What type error are we seeing with this? Locally I don't have any type errors with typing this whole thing as `Meta<Args>`.

Reply:
+1 to Jin, it wasn't a type error but that snapshot issue

Reply:
Alright, installing [@chanzuckerberg/story-utils@^3.0.5](https://github.com/chanzuckerberg/frontend-libs/blob/c6f6de297ddf12fa8dfd4480265a23b472925159/packages/story-utils/CHANGELOG.md#305-2022-05-09) resolves this

Reply:
thanks!!